src/references/damageable_objects.md

```diff
@@ -0,0 +1,93 @@
+---
+id: damageable_reference
+name: Damageable Objects Reference
+title: Damageable Objects Reference
+tags:
+    - Reference
+---
+
+# Damageable Objects
+
+## Summary
+
+The **Damageable Object** is a **Core Object** that implements the **Damageable** interface and has properties, events, and functions for objects which can take damage and die.
```

Comment:
The Damageable interface *is* the properties, events, and functions, right? Is the word "interface" used somewhere specifically? If not I would cut it, and if so, I would define it redundantly, since "interface" can be an abstract and frustrating term to the uninitiated.